gulpfile.js

```diff
@@ -38,7 +38,10 @@ gulp.task('sass', function() {
   return gulp.plumbedSrc(paths.src + '/Layout/Sass/*.scss')
     .pipe(sourcemaps.init())
     .pipe(sass({
-      includePaths: ['./node_modules/bootstrap-sass/assets/stylesheets']
+        includePaths: [
+            './node_modules/bootstrap-sass/assets/stylesheets',
```

Comment:
ja dat kan , de reden waarom ik ze er beide heb gezet is omdat we dan bij het importeren van bootstrap sass het hele path moeten ingeven bij al die files. Omdat dat er toch default heb ik   ze er beide gezet